src/cookies.test.ts

```diff
@@ -2,18 +2,23 @@ import * as _Cookies from 'js-cookie';
 import {
     readGuCookie,
     readIabCookie,
+    readLegacyCookie,
     writeGuCookie,
     writeIabCookie,
+    writeLegacyCookie,
 } from './cookies';
 import {
     GU_COOKIE_NAME,
     GU_COOKIE_VERSION,
     IAB_COOKIE_NAME,
     COOKIE_MAX_AGE,
+    LEGACY_COOKIE_NAME,
 } from './config';
 
 const Cookies = _Cookies;
 
+const OriginalDate = global.Date; // Mock the Date constructor to always return the beginning of time
```

Comment:
Is this comment correct? And do we still need to do this since we're mocking Date.now() only?